pkg/controller/constraint/constraint_controller.go

```diff
@@ -303,8 +300,26 @@ func (r *ReconcileConstraint) Reconcile(ctx context.Context, request reconcile.R
 		status.Status.Errors = nil
 
 		if c, err := r.cfClient.GetConstraint(instance); err != nil || !reflect.DeepEqual(instance, c) {
+			generateVAPB, VAPEnforcementActions, err := shouldGenerateVAPB(*DefaultGenerateVAPB, enforcementAction, instance)
+			if err != nil {
+				status.Status.Errors = append(status.Status.Errors, constraintstatusv1beta1.Error{Message: err.Error()})
+				if err2 := r.writer.Update(ctx, status); err2 != nil {
+					log.Error(err2, "could not get enforcement actions for VAP")
```

Comment:
This error is when updating constraint status failed right? please update the log message.